.github/workflows/php-tests-wp-4-7-php-5-6.yml

```diff
@@ -1,11 +1,30 @@
 name: PHP Tests
 
+defaults:
+  timeout-minutes: 20
+
 on:
   push:
     branches:
       - develop
       - main
+    # Only run if PHP-related files changed.
+    paths:
+      - '**.php'
+      - 'phpcs.xml.dist'
+      - 'phpmd.xml'
+      - 'phpstan.neon.dist'
+      - 'composer.json'
+      - 'composer.lock'
```

Comment:
We don't have a `phpmd.xml` or `phpstan` config file so let's exclude those.

Also, we should probably add `phpunit*.xml*` to cover dist + multisite variants.